terraform/healthcheck/cloudwatch.tf

```diff
@@ -20,7 +20,7 @@ resource "aws_cloudwatch_metric_alarm" "healthcheck" {
   alarm_description   = "Alarm for ${local.domain_name_base[var.environment]} has been triggered"
   actions_enabled     = "true"
   alarm_actions       = [aws_sns_topic.sns_topic.arn, var.alerter]
-  treat_missing_data  = "breaching"
```

Comment:
just checking what does this do?

Reply:
this is to cover the gap between alarm creation and trigger innovations we do not want a state change at this time